frontend/models/product-list/index.ts

```diff
@@ -59,6 +64,17 @@ export async function findProductList(
          ? await fetchDeviceWiki(productList.deviceTitle)
          : null;
 
+   const algoliaApiKey = createProductListAlgoliaKey({
+      appId: ALGOLIA_APP_ID,
+      apiKey: ALGOLIA_API_KEY,
+      filters:
+         productList.filters && productList.filters.length > 0
```

Comment:
Looks like these filters are already represented in the function being called. It seems like we meant to delete this `filters` param after moving the code to the function but forgot to.

Reply:
> Looks like these filters are already represented in the function being called

@danielbeardsley What do you mean by this?

Reply:
Maybe I'm missing it, but it looks like this exact same expression (`filters: ...`) is in the function being called `createProductListAlgoliaKey()`

Reply:
Oh man, you're right! Now I see what you mean 😅  Fixing it  👨‍💻